src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -0,0 +1,175 @@
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: pv7ps
+  namespace: default
+spec:
+  selector: {}
+  template:
+    metadata: {}
+    spec:
+      bootstrap: {}
+      infrastructureRef:
+        apiVersion: infrastructure.giantswarm.io/v1alpha2
+        kind: AWSMachineDeployment
+        name: pv7ps
+        namespace: default
+      metadata: {}
+```
+
+As with *Cluster* CR, *MachineDeployment* has the reference to infrastructure object. In this case, it is *AWSMachineDeployment*, which represents the Giant Swarm Cluster API implementation of node pool for the AWS platform.
+
+*AWSMachineDeployment* CR object example:
+
+```
+apiVersion: infrastructure.giantswarm.io/v1alpha2
+kind: AWSMachineDeployment
+metadata:
+  labels:
+    aws-operator.giantswarm.io/version: 8.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/machine-deployment: pv7ps
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: pv7ps
+  namespace: default
+spec:
+  nodePool:
+    description: np1
+    machine:
+      dockerVolumeSizeGB: 100
+      kubeletVolumeSizeGB: 100
+    scaling:
+      max: 10
+      min: 3
+  provider:
+    availabilityZones:
+    - eu-central-1a
+    worker:
+      instanceType: m4.xlarge
+```
+
+## How to create the cluster omitting Giant Swarm API?
+
+All the CRs, mentioned above, have the strict spec and important requirements to be considered valid. 
+There is no cluster/node pool CRs validation available in Control Plane Kubernetes of Giant Swarm for now. 
+Therefore, if you create CR with wrong field values, that can result in a broken tenant cluster, which then causes unwanted alerts in Giant Swarm monitoring system.
+That's why we've developed a simple [kubectl gs plugin](https://github.com/giantswarm/kubectl-gs), which helps to template valid CRs.
+
+Plugin supports rendering CRs:
+  - Tenat control-plane (AWS only):
+    - `Cluster` (API version `cluster.x-k8s.io/v1alpha2`)
+    - `AWSCluster` (API version `infrastructure.giantswarm.io/v1alpha2`)
+  - Nodepool (AWS only):
+    - `MachineDeployment` (API version `cluster.x-k8s.io/v1alpha2`)
+    - `AWSMachineDeployment` (API version `infrastructure.giantswarm.io/v1alpha2`)
+  - `AppCatalog`
+  - `App`
+
+The installation procedure is described in [README](https://github.com/giantswarm/kubectl-gs#how-to-install-plugin).
+There is also [document](https://github.com/giantswarm/kubectl-gs/blob/master/docs/template-cluster-cr.md), describing the templating process in details.
+
+As a result of running CRs rendering([sample](https://github.com/giantswarm/kubectl-gs/blob/master/docs/template-cluster-cr.md#example)), user gets *yaml* file with the valid Cluster/Node pool definitions. 
+Now tenant cluster control-plane and node pool can be created just by applying those CRs to the Giant Swarm Kubernetes Control Plane, .e.g. `kubectl create -f <CRs definition file>.yaml`. 
```

Comment:
```suggestion
The tenant cluster can be created by applying the cluster manifest file to the Giant Swarm Kubernetes Control Plane, .e.g. `kubectl create -f <cluster manifest file>.yaml`. 
```

Reply:
Maybe we should use either "definition" or "manifest". Also in the sentence before.